gm4_apple_trees/data/gm4_apple_trees/item_modifiers/count.json

```diff
@@ -8,5 +8,5 @@
     },
     "score": "gm4_apple_data"
   },
-  "add": true
+  "add": false
```

Comment:
This should be `true`. It takes the item count in the player's selected item (should be 1 less than before the interaction), then this count is added to the count on the armor stand's main hand (1), which is then moved to the player's selected slot.